test/e2e/README.md

```diff
@@ -24,11 +24,36 @@ knative-gcp should be added under [knative-gcp e2e test lib](lib).
 
 ## Running E2E Tests on an existing cluster
 
-To run [the e2e tests](../e2e) with `go test` command, you need to have a
-running environment that meets
-[the e2e test environment requirements](#environment-requirements), and you need
-to specify the build tag `e2e`.
+### Prerequisites
```

Comment:
It seems Matt didn't enable the markdown checker for this repo, and clearly the markdown file is not formatted.
Could you please manually run `prettier --write --prose-wrap=always $(find -name '*.md' | grep -v vendor | grep -v .github | grep -v docs/cmd/)` to format this PR?